experiments/need_shield_study.md

```diff
@@ -50,12 +102,30 @@ I need evidence to know the next stage for my idea:
 4. My feature needs some in-context UI with interaction.  It includes some surveys.  That's (sadly) a **Shield Study**.
 
 
+## Shield Studies WebExtensionExperiments 
+
+Shield Studies WebExtensionExperiments are the most expensive, high-touch path to gather data.
+
+Still here?
+
+That means, you have decided that you need to deploy
+ 
+- a realistic feature 
+- with complex interaction instrumentation
+- with complex ending criteria
+- to release users
+- using an addon.
```

Comment:
Shield studies can deploy in pre-release channels, and while they usually are, they don't have to be add-on studies (ex: observational pref-flip studies).